lib/ansible/module_utils/network/common/config.py

```diff
@@ -40,6 +40,10 @@
     re.compile(r"Current configuration : \d+ bytes")
 ])
 
+try:
+    Pattern = re._pattern_type
+except AttributeError:
+    Pattern = re.Pattern
```

Comment:
```suggestion
    Pattern = re.Pattern

```

Reply:
Sorry about the unrelated commits, not sure how I could have let that happen.